docs/references/sdk/frontend-only.mdx

```diff
@@ -1,7 +1,152 @@
+  import { runOnce } from './utils'
+  import { $clerk, $state } from './stores'
+
+  export const createClerkInstance = runOnce(createClerkInstanceInternal)
+
+  async function createClerkInstanceInternal(options) {
+    let clerkJSInstance = window.Clerk
+
+    if (!clerkJSInstance) {
+      await loadClerkJSScript(options)
+
+      if (!window.Clerk) {
+        throw new Error('Failed to download latest ClerkJS.')
+      }
+      clerkJSInstance = window.Clerk
+    }
+
+    if (!$clerk.get()) {
+      $clerk.set(clerkJSInstance)
+    }
+  }
+  ```
+
+  ### Call `window.Clerk.load()`
+
+  By calling [`window.Clerk.load()`](/docs/references/javascript/clerk/clerk#load) the Clerk class is initialized and your SDK now has access to all its functionality.
+
+  ```tsx {{ mark: [18], filename: 'create-clerk-instance.ts' }}
+  import { loadClerkJSScript } from '@clerk/shared/loadClerkJSScript'
+  import { runOnce } from './utils'
+  import { $clerk, $state } from './stores'
+
+  export const createClerkInstance = runOnce(createClerkInstanceInternal)
+
+  async function createClerkInstanceInternal(options) {
+    let clerkJSInstance = window.Clerk
+
+    if (!clerkJSInstance) {
+      // loadClerkJSScript() codepath
+    }
+
+    if (!$clerk.get()) {
+      $clerk.set(clerkJSInstance)
+    }
+
+    await clerkJSInstance.load(options)
+  }
+  ```
+
+  ### Add event listeners
+
+  Expose properties to the internal state (e.g. a `useState` in React) through adding [event listeners](/docs/references/javascript/clerk/clerk#add-listener).
+
+  ```tsx {{ mark: [[20, 27]], filename: 'create-clerk-instance.ts' }}
+  import { loadClerkJSScript } from '@clerk/shared/loadClerkJSScript'
+  import { runOnce } from './utils'
+  import { $clerk, $state } from './stores'
+
+  export const createClerkInstance = runOnce(createClerkInstanceInternal)
+
+  async function createClerkInstanceInternal(options) {
+    let clerkJSInstance = window.Clerk
+
+    if (!clerkJSInstance) {
+      // loadClerkJSScript() codepath
+    }
+
+    await clerkJSInstance.load(options)
+
+    if (!$clerk.get()) {
+      $clerk.set(clerkJSInstance)
+    }
+
+    clerkJSInstance.addListener((resources) => {
+      $state.setKey('client', resources.client)
+      $state.setKey('session', resources.session)
+      $state.setKey('user', resources.user)
+      $state.setKey('organization', resources.organization)
+    })
+  }
+  ```
+
+  ### Add UI components
+
+  Build out the components that your users will utilize in their app. Call the `mount()` function when the component is in view/mounts and the `unmount()` function when the component is unmounted. These functions are described in the [Clerk class components](/docs/references/javascript/clerk/clerk#components).
+
+  Use the idiomatic way of your framework for doing this. If you can abstract these repetitions into a re-usable hook/directive, then do that.
+
+  ```tsx {{ filename: 'SignIn.tsx', prettier: false }}
```

Comment:
I know it's pseudo-code but would it be possible to make this one syntactically valid? Should it have `function ` in front of `onMount` and `onUnMount` perhaps? Then we could remove `prettier: false`